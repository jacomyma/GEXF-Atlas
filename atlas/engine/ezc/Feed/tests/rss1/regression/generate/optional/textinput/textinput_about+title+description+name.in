<?php
return array( 'id' => 'Channel about',
              'title' => 'Feed title',
              'link' => 'Feed link',
              'description' => 'Feed description',
			  'item' => array( array( 'id' => 'Item about',
                                      'title' => 'Item title',
                                      'link' => 'Item link' ) ),
              'textInput' => array( array( 'about' => 'Textinput about',
                                           'title' => 'Textinput title',
                                           'description' => 'Textinput description',
                                           'name' => 'Textinput name' ) ),
            );
?>
