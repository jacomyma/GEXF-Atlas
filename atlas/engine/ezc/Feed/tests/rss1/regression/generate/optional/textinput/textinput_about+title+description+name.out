There is no data for required element '/rdf:RDF/textinput/link'.
